siliconcompiler/package/https.py

```
@@ -121,12 +121,12 @@ def resolve_remote(self) -> None:
             if auth_token:
                 headers['Authorization'] = f'token {auth_token}'
 
-        self.logger.info(f'Downloading {self.name} data from {data_url}')
+        self.logger.info(f'Downloading {self.display_name} data from {data_url}')
 
         response = requests.get(data_url, stream=True, headers=headers)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)